extensions/gdx-tools/src/com/badlogic/gdx/tools/texturepacker/MaxRectsPacker.java

```diff
@@ -129,7 +130,7 @@ private Page packPage (Array<Rect> inputRects) {
 					throw new RuntimeException("Image does not fit within max page width " + settings.maxWidth + paddingMessage + ": "
 						+ rect.name + " " + width + "x" + height);
 				}
-				if (height > maxHeight && (!settings.rotation || width > maxHeight)) {
+				if (height > maxHeight && width > maxHeight) {
```

Comment:
From what I can tell, you don't need to compare width and maxHeight unless rotation is true. I've never used rotation in texture packing, so  I don't know exactly how it works, but I suspect this line should be reverted unless there's some different behavior in this PR regarding pages with different height and width.

Reply:
Hey Tommy.

In line 120, there is a check for `settings.rotation` being true already.

Thus, if we are in line 132, it means `!settings.rotation` is always true.

Currently, the actual check is `height > maxHeight` because `(!settings.rotation || width > maxHeight)` is always true.

Should the check after `&&` be removed altogether then?

Reply:
I didn't see that part in the GitHub preview; it should be fine how you have it then. Approved!